### PR TITLE
docs: include misc features for v1.20

### PR DESCRIPTION
**Pull request by efekarakus**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

File: site/content/docs/commands/env-package.en.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+$ copilot env package [flags]
+```
+
+## What does it do?
+`copilot env package` prints the CloudFormation stack template and configuration used to deploy an environment.
+
+## What are the flags?
+```console
+-a, --app string          Name of the application.
+  --force               Optional. Force update the environment stack template.
```
Comment (Lou1415926) [inline site/content/docs/commands/env-package.en.md]: Do they align alright when rendered into the web page?

File: site/content/docs/commands/env-package.en.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+```console
+$ copilot env package [flags]
+```
+
+## What does it do?
+`copilot env package` prints the CloudFormation stack template and configuration used to deploy an environment.
+
+## What are the flags?
+```console
+-a, --app string          Name of the application.
```
Comment (huanjani) [inline site/content/docs/commands/env-package.en.md:11]: 🤔 Do we need to line these up or does mkdocs magically do it? Other files appear to be lined up...?

File: site/content/docs/commands/svc-show.en.md [hunk 3]
```diff
@@ -10,13 +10,23 @@ $ copilot svc show
...
 
 ```
-  -a, --app string        Name of the application.
-  -h, --help              help for show
-      --json              Optional. Output in JSON format.
-      --manifest string   Optional. Name of the environment in which the service was deployed;
-                          output the manifest file used for that deployment.
-  -n, --name string       Name of the service.
-      --resources         Optional. Show the resources in your service.
+-a, --app string        Name of the application.
```
Comment (huanjani) [inline site/content/docs/commands/svc-show.en.md:13]: Unindenting is fine, but this undoes the changes made in https://github.com/aws/copilot-cli/pull/3784. 🥺 Made "Outputs" -> "Output" for `--json` and `--manifest`, and added wording to `--manifest`

File: site/content/docs/commands/env-package.en.md (hunk 2 above)
Comment (efekarakus) [reply site/content/docs/commands/env-package.en.md:11]: Done!

File: site/content/docs/commands/env-package.en.md (hunk 1 above)
Comment (efekarakus) [reply site/content/docs/commands/env-package.en.md]: Done!

File: site/content/docs/commands/svc-show.en.md (hunk 3 above)
Comment (efekarakus) [reply site/content/docs/commands/svc-show.en.md:13]: Reverted! sorry I didn't even realize, I blindly copy pasted from the cli's help menu :P

File: site/content/docs/manifest/environment.en.md [hunk 4]
```diff
@@ -167,9 +167,26 @@ List of [AWS Certificate Manager certificate](https://docs.aws.amazon.com/acm/la
 By attaching public or private certificates to your load balancer, you can associate your Backend Services with a domain name and reach them with HTTPS.
 See the [Developing/Domains](../developing/domain.en.md#use-domain-in-your-existing-validated-certificates) guide to learn more about how to redeploy services using [`http.alias`](./backend-service.en.md#http-alias).
 
-<span class="parent-field">http.private.</span><a id="http-private-subnets" href="#http-private-subnets" class="field">`subnets`</a> <span class="type">Array of Strings</span>  
+<span class="parent-field">http.private.</span><a id="http-private-subnets" href="#http-private-subnets" class="field">`subnets`</a> <span class="type">Array of Strings</span>   
 The subnet IDs to place the internal load balancer in.
 
+<span class="parent-field">http.private.</span><a id="http-private-security-groups" href="#http-private-security-groups" class="field">`security_groups`</a> <span class="type">Map</span>    
+Additional security groups to add to the internal load balancer.
```
Comment (Lou1415926) [inline site/content/docs/manifest/environment.en.md]: They are not necessarily "additional" - this field could be just configuring the sg that we put in front of the internal ALB. E.g. `security_groups.ingress.from_vpc` is adding the ingress, instead of adding an sg that allows that ingress (iirc).

Perhaps we can reword it to `Configure security groups to add to the internal load balancer.`?

Comment (efekarakus) [reply site/content/docs/manifest/environment.en.md]: Done!

**Review by Lou1415926 — APPROVED** [review]

**Review by huanjani — APPROVED** [review]
🐿️
